public/js/index.js

```diff
@@ -479,8 +487,7 @@ function initCommentForm() {
                     "clear",
                     $listMenu.data('issue-id'),
                     ""
-                );
-                $listMenu.data('action', 'update'); // Update to reload the page when we updated items
+                ).then(reload);
```

Comment:
This was the only way to make it reliably show the correct status entry on the issue. Without this change, I was unable to assign a issue at all (clearing did work thought).